nilearn/datasets/struct.py

```diff
@@ -106,38 +114,134 @@ def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
     return Bunch(**params)
 
 
-def load_mni152_template():
-    """Load skullstripped 2mm version of the MNI152 originally distributed
-    with FSL.
+def load_mni152_template(resolution=2):
```

Comment:
Sorry if this has been decided already, but I don't remember whether we chose to deprecate the default value of 2mm to 1mm or not... :thinking:  
If we wish to use the 1mm resolution template as the default in the future, we should start deprecating now.
WDYT?

Reply:
Indeed, I don't think we have discussed the deprecation cycle of the default value of the resolution param. We set it equal to 2mm in order not to break many examples. Yet, the functions `compute_brain_mask` and associated (e.g. `compute_multi_brain_mask`) use now resolution=1mm. I let @jeromedockes and/or @bthirion answer this.

Reply:
Since the 1mm is now automatically downloaded, I'm in favor of deprecation of the 2mm for the default value.